openedx/features/survey_report/admin.py

```diff
@@ -0,0 +1,52 @@
+"""
+Django Admin page for SurveyReport.
+"""
+
+
+from django.contrib import admin
+from .models import SurveyReport
```

Comment:
can you use absolute imports?

Reply:
I don't know if is a good idea, I've checked the imports in the other features and they are using relative imports maybe because they are working like external django apps, and maybe we are thinking on move this to another place, but I don't know.

Reply:
Both are ok according to [this](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/django.html?highlight=imports#imports) style guide. But as I see it, it's easier to understand absolute paths when importing from more nested files -just preferences-.